docs/receivers.md

```diff
@@ -10,6 +10,7 @@ Below is a list of supported receivers with links to their documentation pages.
 | Apache Flink Metrics Receiver               | [flinkmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.62.0/receiver/flinkmetricsreceiver/README.md) |
 | Apache Web Server Receiver                  | [apachereceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.62.0/receiver/apachereceiver/README.md) |
 | Apache ZooKeeper Receiver                   | [zookeeperreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.62.0/receiver/zookeeperreceiver/README.md) |
+| AWS CloudWatch Receiver                     | [awscloudwatchreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.62.0/receiver/awscloudwatchreceiver/README.md) |
```

Comment:
nit: I think this should come after AWS CloudWatch Containers alphabetically.

Reply:
I was going by the "Alphabetical ignoring the word Receiver" route. If you feel strongly or others agree I can change